qa/integration-tests/src/test/java/io/camunda/it/migration/MigrationUserTaskUpdateIT.java

```diff
@@ -35,7 +35,7 @@ public class MigrationUserTaskUpdateIT {
   @RegisterExtension
   static final MigrationITInvocationProvider PROVIDER =
       new MigrationITInvocationProvider()
-          .withDatabaseTypes(DatabaseType.ELASTICSEARCH, DatabaseType.OPENSEARCH)
```

Comment:
I need to temporary disable Opesearch test, until a SNAPSHOT version of Tasklist is deployed with the fix in `MigrationRunner` above